src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/UserProfile.html

```diff
@@ -20,6 +18,8 @@
             </span>
         </div>
 
+        <adh-user-profile-image-edit data-path="{{path}}"></adh-user-profile-image-edit>
+
```

Comment:
slightly unrelated: the text "edit profile pic" should be in inverted text color.